.github/workflows/label_app_preview.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2023 Sharezone UG (haftungsbeschränkt)
+# Licensed under the EUPL-1.2-or-later.
+#
+# You may obtain a copy of the Licence at:
+# https://joinup.ec.europa.eu/software/page/eupl
+#
+# SPDX-License-Identifier: EUPL-1.2
+
+# This is a GitHub Actions workflow that triggers a Codemagic build when a PR is
+# labeled with "build-app-preview". It posts a comment to the PR with links and
+# qr codes to the app preview.
+
+name: App Preview
+
+on:
+  pull_request:
+    types:
+      - labeled
+      - opened
+      - synchronize
+
+jobs:
+  label_app_preview:
+    # Only run this job if the PR is labeled with "build-app-preview".
+    #
+    # Keep in mind that a new build will be triggered when the PR is labeled
+    # with any lable as long as the label "build-app-preview" is included in the
+    # list of labels. For example, if the PR is labeled with "build-app-preview"
+    # and "bug", the job will be triggered when the label "bug" is removed.
+    if: contains(github.event.pull_request.labels.*.name, 'build-app-preview')
+    runs-on: ubuntu-latest
```

Comment:
So if labels are changed then this is retriggered?

Reply:
Yes